sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/models/ImageLocation.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.implementation.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** An image response item that provides a URL from which an image may be accessed. */
+@Immutable
+public final class ImageLocation {
```

Comment:
This should be a public class

Reply:
The [public class](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ImageLocation.java) is not same as tsp model https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/images/images.tsp#L62-L65

Hence, I configured in tspconfig to move this class to implementation package.

If we think we should "sync" our public ImageLocation to this, I can remove that configure.